static/index.html

```diff
@@ -0,0 +1,81 @@
+<html>
```

Comment:
can generate this file dynamically using htmlWebpackplugin. and include all the vendors there itself.